notebooks/data_viz_easy/raw/ex1.ipynb

```diff
@@ -423,87 +179,32 @@
    "source": [
     "## Step 3: Visualize the data \n",
     "\n",
```

Comment:
I was torn on this.  People should make an initial graphic to understand their data, and this is the perfect graph to make in this circumstance.

On the flip side, it's more interesting if we build the steps into the scenario.  Some example ways to phrase the question
1. The museum board wants an overview of the data. Create a plot that ...
2. The firehouse museum claims they ran an event in 2014 that brought an incredible number of visitors, and that they should get extra budget to run a similar event again. The other museums think these types of events aren't that important, and budgets should be split purely based on recent visitors on an average day.  To show the museum board how the event compard to regular traffic at each museum, create a plot of ...